webapp/src/main/java/org/opentestsystem/rdw/admin/service/SchoolService.java

```diff
@@ -19,4 +20,12 @@
      */
     Set<Organization> getSchools(@NotNull User user);
 
+    /**
+     * Checks if a school exists by the given permission scope.
+     *
+     * @param permissionScope  the permission scope to check
+     * @param schoolId          the school id to check
+     * @return true if the school exists and is part of the permission scope.
+     */
+    boolean existsByPermissionScope(@NotNull PermissionScope permissionScope, int schoolId);
```

Comment:
Do we really need `ByPermissionScope` in the method name? Seems pretty obvious from the required parameter.